Sources/Plasma/PubUtilLib/plPhysX/plPXSimulation.cpp

```diff
@@ -598,6 +598,8 @@ physx::PxScene* plPXSimulation::InitSubworld(const plKey& world)
     }
 #endif
 
+    scene->setDominanceGroupPair(plSimDefs::kGroupAvatar, plSimDefs::kGroupDynamic, { 0, 1 });
```

Comment:
What's the significance here of `{0, 1}`?

Reply:
It's a `physx::PxDominanceGroupPair`. The usage of it is a little strange, though... You specify either 0 or 1 to indicate the dominance, but they act sort of like a multplier. So in a collision between a cone and an avatar, a cone imparts no force onto an avatar, but the avatar imparts its force unchanged onto a cone.

Reply:
Ah, that seems pretty straightforward.